src/components/App.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  handleClick = (btn) => {
+    const data = this.state;
+    const result = calculate(data, btn);
+    this.setState(result);
+    console.log(this.state);
+  }
+
```

Comment:
- [ ] Kindly remove the console.log statement here. Debugging code shouldn't be pushed to production.